### ISSUE TITLE
Ambiguity in understanding for 1.3.3 sensory characteristics

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#750]
While not ideal, 1.3.3 only normatively refers to *instructions* that refer to things by their shape/size/etc. 1.3.3 does not fail (at least to my understanding) for content/controls that themselves rely on shape/size/etc to be understood, without any instructions being around to refer to them. (For problems relating to these in isolation, they'd probably fall under 1.1.1, 1.3.1 and/or 4.1.2).

However, the understanding text for 1.3.3 is slightly unclear/unfocused on this aspect. https://www.w3.org/WAI/WCAG21/Understanding/sensory-characteristics.html

> Some users with disabilities are not able to perceive shape or position due to the nature of the assistive technologies they use. This Success Criterion requires that additional information be provided to clarify anything that is dependent on this kind of information.

This seems to imply that controls/information conveyed through shape or position need additional information themselves, when in fact this is (I think) about the need for *instructions* that refer to these needing to have additional information.

> Some users with disabilities are not able to perceive shape or position due to the nature of the assistive technologies they use. This Success Criterion requires that additional information be *given as part of any instructions provided for understanding and operating content* to clarify anything that is dependent on this kind of information.

Similarly, in benefits:

> People who are blind and people who have low vision may not be able to understand information if it is conveyed by shape and/or location. Providing additional information other than shape and/or location will allow them to understand the information conveyed by shape and/or alone.

(which also seems to end rather abruptly/awkwardly) should ideally be

> People who are blind and people who have low vision may not be able to understand *instructions if they rely only on a description of the shape and/or location of content*. Providing additional information *in any instructions* other than shape and/or location will allow *users* to understand the *instructions even if they cannot perceive shape and/or location*.

Lastly, the first example provided seems to be completely beside the point.

> Example 1: A schedule of competitive events uses color and shape to distinguish the time of each event
> A table presents a list of times across the top row and a list of events in the first vertical column. The cell corresponding to the time of a particular event has a specific background color and diamond shaped glyph so it can be identified by color and shape.

This talks about problems which seem related to 1.4.1 Use of Color, and of the diamond glyphs that would arguably fall under 1.1.1 Text Alternatives ... it does not make any reference to *instructions*, which this SC applies to.

Happy to make a PR if it's agreed that this is an issue...

**Comment by awkawk** [comment]
@patrickhlauke I don't think that the changes for the intent or the benefits are needed, but the first example is off. If the example read I think it would be better:

Example 1: Instructions for interpreting a schedule of competitive events references colored icons in different shapes to indicate the venue for each event. A table presents a list of times across the top row and a list of events in the first vertical column and instructions are provided under the table: "Events marked with a blue diamond are played on field A and events marked with a green circle are played on field B." The instructions rely on color and shape only and result in a failure of this criterion.

Related to this, F26 (https://www.w3.org/TR/WCAG20-TECHS/F26.html) seems afflicted by this same problem.

**Comment by patrickhlauke** [comment]
@awkawk changes may not be absolutely necessary, but would make it crystal clear even on skim reading (and that "will allow them to understand the information conveyed by shape and/or alone" ending for the benefit doesn't look right, regardless)

And indeed, F26 does seem like a 1.1.1 failure, not a 1.3.3 one.

**Comment by alastc** [comment]
The benefits bit is at least missing the word 'location' from the last couple of words. 

I wonder if there was a bit of 'constructive ambiguity' to nudge people to not rely on color/shape at all? 
But including 'instructions' does seem reasonable, if you can make a PR Patrick?

**Comment by patrickhlauke** [comment]
some further thoughts on 1.3.3 ... does a very strict reading of 1.3.3 completely disallow the use of any symbols/graphical shapes in isolation (let's assume they have correct alternative text, so not a 1.1.1 failure, etc).

If i have a table, that uses icons (say a checkmark, a cross, and a question mark) ... that's fine. But if i now add a legend that shows the checkmark meaning supported, cross meaning not supported, and question mark meaning unknown...the legend here would act as an "instruction provided for understanding", and as it relies on the user being able to recognise/differentiate the shapes of these icons, would it fail 1.3.3? if so...this seems harsh, but also odd (as then NOT having a legend would make it pass).

**Comment by detlevhfischer** [comment]
This is a minefield you have gotten us into, Patrick, can‘t you be content with a fudge? (think of a smiley here) 
I think this is indeed a fudge exposed - and Alastair is probably right that there was a bit of constructive ambiguity to nudge people in the direction of not relying on sensory characteristics alone, independent of instructions. Not sure whether the benefit of that will outweigh the confusion.

**Comment by JAWS-test** [comment]
> If i have a table, that uses icons (say a checkmark, a cross, and a question mark) ... that's fine. But if i now add a legend that shows the meaning of each icon (the checkmark meaning supported, cross meaning not supported, and question mark meaning unknow), the legend here would act as an "instruction provided for understanding", and as it relies on the user being able to recognise/differentiate the shapes of these icons, would it fail 1.3.3? if so...this seems harsh, but also odd (as then NOT having a legend would make it pass).

I don't see any problem with that. If the content is understandable without a legend, the graphical elements need alternative texts according to SC 1.1.1 and the table is therefore understandable for everyone. However, if a legend is necessary, it is not sufficient if the legend only refers to the visual properties of the graphical elements (because blind users cannot perceive them), but the legend must also refer to the alternative texts. So 1.3.3 is only valid if there is a legend -  because 1.3.3 is a SC which handles legends and similar elements.

**Comment by alastc** [comment]
> If i have a table that uses icons (say a checkmark, a cross, and a question mark) ... that's fine. But if i now add a legend .. would it fail 1.3.3? 

It isn't clear from the doc, but isn't alt text a viable technique? (E.g. F26 references H37- alt text on images.) And as @JAWS-test said, if the alt texts between legend and items match up, that should work.

So in combination with 1.1.1 & 1.4.11, that scenario should pass IMHO.

**Comment by patrickhlauke** [comment]
fair enough, i clearly stared at this too long (but perhaps this is an indication that more clarification is needed somewhere so this sort of misunderstanding doesn't happen...i'll have a further think - and again, this is brought on by F26 perhaps not making it crystal clear here)

**Comment by JAWS-test** [comment]
@patrickhlauke Please remove or revise F26 and change the Understanding, then it will be easier for everyone to understand. So far, it has indeed been wrongly formulated.

**Comment by electronicwoft** [comment]
Is a glyph the same as a symbol? Is there a normative definition of either of these two concepts? Is a standalone pictogram acceptible whereas a standalone ideo or logogram are not? This understanding section should talk about the sensory characteristics of graphical content more broadly if for no other reason than to make it abundantly clear that the scope of success criterion 1.3.3 covers only information that assists users in understanding and operating content is textual. In my view, there is a requirement for Future WCAG to cover iconography.

**Comment by grifare** [comment]
I personally think F26 itself is more confusing than the use of graphical symbols.
Here is the list of issues with it:

- The title "Failure of Success Criterion 1.3.3 due to using a graphical symbol alone to convey information" suggests that using an asterisk without a legend to denote required fields would fail, which is misleading.
- The description section can be interpreted to mean that the mere existence of any graphical symbol used without a legend would fail. The example states that graphical symbols would fail only if there are instructions referring to them.
- [patrickhlauke](https://github.com/patrickhlauke)
[on Oct 7, 2019](https://github.com/w3c/wcag/issues/750#issuecomment-539116149) · explained how confusing it sounds now. I am puzzled by how the presence of legends can cause a failure.
- There is only one example provided, which is not a common scenario on websites. It raises the question of whether people cannot differentiate circles from squares. An explicit example of whether asterisks alone on forms to denote required fields would fail or not would be helpful.
- As of now (2025), there seems to be no clarification or updates to the F26 page.
- The WCAG text should be free from errors such as "An square."

![Image](https://github.com/user-attachments/assets/80452a65-800d-45ad-938e-dc8f3105dde8)
